service/controller/v8/version_bundle.go

```diff
@@ -8,31 +8,46 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
+				Component:   "kubernetes",
+				Description: "Update kubernetes to 1.14.3. More info here: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md",
+				Kind:        versionbundle.KindChanged,
+			},
+			{
+				Component:   "calico",
+				Description: "Update calico to 3.7.2. More info here: https://docs.projectcalico.org/v3.7/release-notes/",
+				Kind:        versionbundle.KindChanged,
+			},
+			{
+				Component:   "containerlinux",
+				Description: "Updated to 2079.5.1. More info here: https://github.com/coreos/manifest/releases/tag/v2079.5.1",
```

Comment:
The speech is different. Above is `Update` and here is `Updated`. @rossf7 since we had this yesterday already, how should it be now? Apologies for asking again. 🙈

Reply:
Copied pasted from also the other operators. if it must be changed then I'll go to change everwhere.

Reply:
Autopiloted through this to fast - well spotted.
It should be past tense AFAIK, so `Updated`.

Reply:
Going to change for that.

Reply:
It should be imperative so `Update`. `fmt` is outdated doing a PR now.

Reply:
ouch 👍

Reply:
See https://github.com/giantswarm/fmt/pull/55.

Reply:
Thank you @rossf7. I've changed it ;-)